frontend/src/__tests__/cypress/cypress/tests/e2e/dataScienceProjects/connections/testDataConnectionCreation.cy.ts

```diff
@@ -1,106 +0,0 @@
-import { HTPASSWD_CLUSTER_ADMIN_USER } from '~/__tests__/cypress/cypress/utils/e2eUsers';
-import { projectListPage, projectDetails } from '~/__tests__/cypress/cypress/pages/projects';
-import { deleteOpenShiftProject } from '~/__tests__/cypress/cypress/utils/oc_commands/project';
-import type { DataScienceProjectData, AWSS3BucketDetails } from '~/__tests__/cypress/cypress/types';
-import { connectionsPage, addConnectionModal } from '~/__tests__/cypress/cypress/pages/connections';
-import { loadDSPFixture } from '~/__tests__/cypress/cypress/utils/dataLoader';
-import { createCleanProject } from '~/__tests__/cypress/cypress/utils/projectChecker';
-import { deleteModal } from '~/__tests__/cypress/cypress/pages/components/DeleteModal';
-import { AWS_BUCKETS } from '~/__tests__/cypress/cypress/utils/s3Buckets';
-import {
-  retryableBefore,
-  wasSetupPerformed,
-} from '~/__tests__/cypress/cypress/utils/retryableHooks';
-
-describe('Verify Data Connections - Creation and Deletion', () => {
```

Comment:
I might be wrong here but I think this is actually testing creating Connections in a project, it's just misnamed as Data Connection. I don't think we should delete it, but instead remove the name `Data` from anything

Reply:
According to @antowaddle we shouldn't delete it and just rename anything to Connections